test/test_wordcloud.py

```diff
@@ -229,6 +230,18 @@ def test_mask_contour():
     assert_true(all(sm_array[100, 300] == [0, 0, 255]))
 
 
+def test_mask_contour_transparent():
+    # test if mask contour can be drawn on a transparent background
+    # https://github.com/amueller/word_cloud/pull/389
+    examples_dir = os.path.realpath(os.path.join(os.path.dirname(__file__), '..', 'examples'))
+    mask_file = os.path.join(examples_dir, 'alice_mask.png')
+    mask = np.array(Image.open(mask_file))
+    w = WordCloud(mode='RGBA', background_color=None, contour_color='blue', contour_width=2, mask=mask,
+                  max_words=50, width=800, height=800)
+    w.generate(THIS)
```

Comment:
So does this give an error? You could probaby also just create an empty mask to reproduce it instead of loading one, right?

Reply:
I don't know what you describe works, but why would you be trying to draw a contour with an empty mask?

Reply:
Right maybe that's not the very best idea but you could just add a stripe to one side or something. The point is to make the test as simple and self-contained as possible. And if you can reproduce the error with a small numpy array that would be easier.

Reply:
The image is in the repo, so it is adequately self-contained. Sure, someone with a better grasp of PIL can remove this dependency in the future.

Reply:
```python
mask = np.ones((100, 100))
mask[:, -10:] = 0
```
should do it, right?

Reply:
Actually better
```
mask = np.zeros((100, 100, 3), dtype="ubyte")
mask[:, -10:] = 255
```

Reply:
Yup! Looks good. This would have taken me at least another 30' to figure out. Haven't worked with matrices and stuff for quite a long time.